### ISSUE TITLE
Feedback

### ISSUE THREAD

**Issue by Drimdave** [CodeZhifty/voiceassistant#2]
Hi bro,
Is it possible to change the language being spoken by Eva? 
Also, can multiple languages be added?

**Comment by zhiftyDK** [comment]
@Drimdave Yes it is possible if you look at "voices google speechsynthesis api" on google

**Comment by zhiftyDK** [comment]
@Drimdave You can also do it like this 

//Get and apply voices to the speak function
let voices;
window.speechSynthesis.addEventListener("voiceschanged", () => {
    voices = window.speechSynthesis.getVoices();
});
function speak(input) {
    var utterance = new SpeechSynthesisUtterance(input);
    utterance.voice = voices[6];
    window.speechSynthesis.speak(utterance);
}

**Comment by Drimdave** [comment]
Thank you so much.
